volumina/volumeEditorWidget.py

```diff
@@ -245,6 +245,12 @@ def onShapeChanged():
                 cropMin = (self.editor.posModel.time,self.editor.cropModel._crop_extents[0][0],self.editor.cropModel._crop_extents[1][0],self.editor.cropModel._crop_extents[2][0],0)
                 self.quadViewStatusBar.updateShape5Dcropped(cropMin,self.editor.posModel.shape5D)
 
+            timeDim = self.editor.posModel.shape5D[0]
```

Comment:
I'd rather have both the slider and the time spin box enabled/disabled in the same function, whichever that is.